content/docs/command-reference/import-url.md

```diff
@@ -88,6 +88,15 @@ Specific explanations:
   running. DVC automatically expands this URL into a regular S3, SSH, GS, etc
   URL by appending `/path/to/file` to the `myremote`'s configured base path.
 
+The `deps` field in the generated `.dvc` file contains a hash value along with
+the `path` to track changes in the remote file/directory. The hash type depends
+on the type of remote location (protocol) being tracked. For instance,
+files/directories tracked from HTTP(s)/S3/Azure/GS remote locations will have an
+[ETag](https://en.wikipedia.org/wiki/HTTP_ETag) value in the `deps` field, while
+files/directories tracked from SSH/local locations will have an `md5` hash
+value. Files from HDFS remote locations use a `checksum` key in the `deps`
+field.
+
```

Comment:
Ah thanks! But the intention of the ticket wasn't to repeat all this info here! Just double check that any existing mentions of hash values are correct. We should link "`.dvc` files" to the corresponding user guide though, so that people can see the full spec there, including types of hash values.

Reply:
Also in line 111 can you capitalize the first word? "It". Thanks

Reply:
I see this link containing the info we need but how can I link precisely that paragraph which talks about the different hash values instead of the section `dvc.lock file`?
https://dvc.org/doc/user-guide/dvc-files-and-directories#dvclock-file

It would be great if we could link this paragraph:
> Regular dependencies and all kinds of outputs (including metrics and plots files) are also listed (per stage) in dvc.lock, but with an additional field to store the hash value of each file or directory tracked by DVC: md5 for local file system dependencies and SSH external dependencies, etag for HTTP, S3, Azure, and Google Cloud, and checksum for HDFS.

Reply:
Can't link to a paragraph haha. That's not necessary, the anchor is enough... People have eyes 😉

Reply:
This p is still there... Updated https://github.com/iterative/dvc.org/pull/1494#pullrequestreview-442823296. Resolving here.